libopenarc/Makefile.am

```diff
@@ -8,7 +8,7 @@ endif
 LDADD = ./libopenarc.la
 
 lib_LTLIBRARIES = libopenarc.la
-libopenarc_la_SOURCES = base64.c arc.c arc.h arc-canon.c arc-canon.h arc-dns.c arc-dns.h arc-internal.h arc-keys.c arc-keys.h arc-tables.c arc-tables.h arc-types.h arc-util.c arc-util.h
+libopenarc_la_SOURCES = base64.c base64.h arc.c arc.h arc-canon.c arc-canon.h arc-dns.c arc-dns.h arc-internal.h arc-keys.c arc-keys.h arc-tables.c arc-tables.h arc-types.h arc-util.c arc-util.h
```

Comment:
If base64.h isn't in SOURCES it won't be included in the dist tarball, no matter how you invoke autotools. It's not necessary for building from a git checkout, but that's not the only way to do things.